client/src/components/screens/PatientSubmission/components/ClinicalInformation/index.jsx

```diff
@@ -408,12 +407,16 @@ class ClinicalInformation extends React.Component {
 
     const AnalysisTestCodesValues = Object.values(AnalysisTestCodes)
 
-    const initialAnalysisValue = get(localStore, 'serviceRequest.code', undefined);
-    let initialAnalysisNote = get(localStore, 'serviceRequest.note', undefined);
-    const initialInterpretation = get(localStore, 'cgh.interpretation', undefined);
-    const initialPrecision = get(localStore, 'cgh.precision', undefined);
+    const initialAnalysisValue = get(serviceRequest, 'code', undefined);
```

Comment:
I think we should do: `serviceRequest?code` ... it will be undefined if it defaults to nothing. I think we should use lodash only for highly specialized stuff. (comment valid for the lines below)